tests/unit/synapseclient/core/unit_test_utils.py

```diff
@@ -100,6 +100,37 @@ def __init__(self, id_attr_name, id):
         assert utils.id_of(foo) == "123"
 
 
+# TODO: Add a test for is_synapse_id_str(...)
```

Comment:
TODO

Reply:
Hi Bryan could you elaborate on this comment?

Reply:
You added a # TODO comment. Is this something you will plan to accomplish this this PR?

Reply:
No, I was going to make a ticket and add it to the Tech Debt Epic on Jira. I placed the TODO here so it's easier to see where the unit test should go.

Reply:
Makes sense. When you create the Jira can you add it to the TODO here